app/model_util.py

```diff
@@ -57,9 +57,10 @@ def create_tables(retry=5, add=True):
             print(f"Creating tables on database {conn.url.database}")
             models.Base.metadata.create_all()
 
-            print("Refreshed tables:")
-            for m in models.Base.metadata.tables:
-                print(f" - {m}")
+            table_list = ", ".join(list(models.Base.metadata.tables.keys()))
+            print(f"Refreshed tables: {table_list}")
+            # for m in models.Base.metadata.tables:
```

Comment:
Do you mean to leave the print statements in?

Reply:
Yeah. I'm replacing one print statement that is pretty long, with a more condensed one.